ui/lib/config-ui/addon/components/login-settings/page/list.hbs

```diff
@@ -0,0 +1,38 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<PageHeader as |p|>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      UI Login Rules
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+{{! currently here empty per design, TBD if actions or filters will be added }}
```

Comment:
The designs don't include it, but if you'd like to make a ticket and add a stretch goal to include a search input to filter the list by name - you're welcome to! 😄